recipes/tz/all/test_package/conanfile.py

```diff
@@ -14,14 +15,19 @@ class TzTestConan(ConanFile):
     def _settings_build(self):
         return getattr(self, "settings_build", self.settings)
 
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
     def build_requirements(self):
-        self.tool_requires(self.tested_reference_str)
         if self._settings_build.os == "Windows" and not self.conf.get("tools.microsoft.bash:path", check_type=str):
             self.tool_requires("msys2/cci.latest")
 
     def generate(self):
         # INFO: zdump does not consume TZDATA, need to pass absolute path of the zoneinfo directory
-        self.tzdata = self.dependencies.build['tz'].buildenv_info.vars(self).get('TZDATA')
+        self.tzdata = self.dependencies['tz'].runenv_info.vars(self).get('TZDATA')
         with open("tzdata.info", "w") as fd:
             fd.write(self.tzdata)
```

Comment:
Fails with Conan v1 otherwise.
```suggestion
        tzdata = self.dependencies['tz'].runenv_info.vars(self).get('TZDATA')
        save(self, "tzdata_path", tzdata)
        save(self, "with_binary_db", str(self.dependencies['tz'].options.with_binary_db))
```